
# [CVE-2020-24372](https://github.com/LuaJIT/LuaJIT/issues/603)

## Description

- `LuaJIT through 2.1.0-beta3 has an out-of-bounds read in lj_err_run in lj_err.c.`

## Cvss Data

- **Access Vector**:
  - NETWORK
- **Base Score**:
  - 5.0

## Scores

- **Exploitability Score**:
  - 10.0
- **Impact Score**:
  - 2.9
- **Base Severity**:
  - MEDIUM

## Other Information

- **Publish Date**:
  - 2020-08-17 17:15:13
- **Vulnerability Status**:
  - Analyzed

## References

- **cve@mitre.org**: https://github.com/LuaJIT/LuaJIT/issues/603
